environment/src/main/java/com/sequenceiq/environment/environment/service/EnvironmentCreationService.java

```diff
@@ -70,7 +70,7 @@ public EnvironmentDto create(EnvironmentCreationDto creationDto) {
     private Environment initializeEnvironment(EnvironmentCreationDto creationDto) {
         Environment environment = environmentDtoConverter.creationDtoToEnvironment(creationDto);
         environment.setResourceCrn(createCrn(creationDto.getAccountId()));
-        environment.setProxyConfigs(proxyConfigService.findByNamesInAccount(creationDto.getProxyNames(), creationDto.getAccountId()));
+        //environment.setProxyConfigs(proxyConfigService.findByNamesInAccount(creationDto.getProxyNames(), creationDto.getAccountId()));
```

Comment:
do we need these lines? if not, please remove

Reply:
ok